crates/ruff_cli/tests/snapshots/show_settings__display_default_settings.snap

```diff
@@ -0,0 +1,361 @@
+---
+source: crates/ruff_cli/tests/show_settings.rs
+info:
+  program: ruff
+  args:
+    - check
+    - "--show-settings"
+    - unformatted.py
+---
+success: true
+exit_code: 0
+----- stdout -----
+Resolved settings for: "[BASEPATH]/crates/ruff_cli/resources/test/fixtures/unformatted.py"
+Settings path: "[BASEPATH]/pyproject.toml"
+cache_dir = "[BASEPATH]/.ruff_cache"
```

Comment:
Nit: might be nice to visually separate the two lines above (which are meta-information about the settings) from the settings themselves. Maybe adding a comment above the `cache_dir` line, like `Top-level Settings` or similar? Just an idea, open to better solutions.

Reply:
Good idea! I might do something like `# General Settings` if that works.